src/styles/style.css

```diff
@@ -0,0 +1,387 @@
+  line-height: 24px;
+}
+.newsletter {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 24px 24px 0px 24px;
+  text-align: center;
+  height: 230px;
+  background-color: var(--blue-100);
+}
+.newsletter_title {
+  font-family: "Montserrat", sans-serif;
+  font-style: normal;
+  color: var(--gray-800);
+  font-weight: 500;
+  font-size: 16px;
+  margin-bottom: 16px;
+  line-height: 20px;
+  height: 20px;
+}
+.newsletter_paragraph {
+  font-weight: 400;
+  font-family: "Lato", sans-serif;
+  color: var(--gray-800);
+  font-size: 12px;
+  margin-bottom: 25px;
+  line-height: 14px;
+  width: 327px;
+  height: 28px;
+}
+.footer {
+  text-align: center;
+  padding-top: 34px;
+  height: 258px;
+  background-color: var(--blue-900);
+}
+.footer_list {
+  font-size: 25px;
+  color: var(--gray-100);
+  padding-bottom: 18px;
+  letter-spacing: 10px;
+}
+.footer_list a {
+  color: var(--gray-100);
+}
+.author {
+  font-family: "Lato", sans-serif;
+  font-size: 14px;
+  color: var(--gray-200);
+  padding-bottom: 20px;
+}
+.footer_links a {
+  font-family: "Lato", sans-serif;
+  color: var(--blue-200);
+  font-size: 12px;
+  padding: 0 16px;
+}
+.form {
+  max-width: 19rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  position: relative;
+}
+.form_email {
+  flex: 1;
+  padding: 16px 13px 16px 13px;
+  border-radius: 5px 0px 0px 5px;
+  font-size: 12px;
+  color: var(--gray-900);
+  border: none;
+}
+.form_button {
+  display: flex;
+  width: 5rem;
+  background: var(--orange-300);
+  padding: 18px 100px 12px 13px;
+  border: none;
+  border-radius: 0px 5px 5px 0px;
+  font-size: 12px;
+  color: var(--orange-900);
+  cursor: pointer;
+}
+.form_button img {
+  margin-top: -5px;
+  display: flex;
+  padding: 0 7px;
+}
+.text {
+  margin-top: 5px;
+  width: 285px;
+  font-size: 18px;
+  border-radius: 5px;
+  background-color: var(--blue-100);
+  font-family: "Lato", sans-serif;
+  font-weight: 700;
+}
+.text_true {
+  color: #eee;
+  background-color: #009933;
+}
+.text_false {
+  color: #eee;
+  background-color: #c9223a;
+}
+
+@media only screen and (min-width: 834px) {
+  .main {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 48px 0px 128px 0px;
+  }
+  .logo {
+    margin-left: 18px;
+  }
+  .nav ul {
+    padding-right: 64px;
+  }
+  .images {
+    display: flex;
+    gap: 24px;
+  }
+  .smaller_images {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+  }
+  .image_primary {
+    width: 468px;
+    height: 274px;
+    background-color: var(--gray-200);
+    border-radius: 8px;
+    margin-bottom: 80px;
+    margin-top: 48px;
+  }
+  .smaller_images {
+    margin-top: 48px;
+  }
+  .image_1 {
+    width: 214px;
+    height: 125px;
+    background-color: var(--gray-200);
+    border-radius: 8px;
+  }
+  .image_2 {
+    width: 214px;
+    height: 125px;
+    background-color: var(--gray-200);
+    border-radius: 8px;
+  }
```

Comment:
Avoid giving class names with appendix numbers, they are quite ambiguous, in this case the choice is even more trivial since the images have the same classes, so you could just use `image` class.